modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -1070,7 +1070,7 @@ CV__DNN_INLINE_NS_BEGIN
      *  backend.
      */
     CV_EXPORTS_W
-    Net readNetFromModelOptimizer(const String &xml, const String &bin);
+    Net readNetFromModelOptimizer(const String &xml, const String &bin = "");
```

Comment:
`= String()` / `std::string()`

to avoid processing of unnecessary extra buffers

(other APIs have similar problem so it is better to resolve that in a separate PR)